src/adagrams.js

```diff
@@ -1,15 +1,187 @@
 export const drawLetters = () => {
-  // Implement this method for wave 1
+  let letterPool = [
```

Comment:
I kind of like the blunt solution of just having multiple copies of each letter.

This should be `const` though, since you aren't going to re-assign `letterPool`.

```suggestion
  const letterPool = [
```

`const` is very minimal protection (it doesn't prevent modifying the array, just _reassigning_ the variable) but it's better than nothing so should be used when possible.

Best practice is declaring everything as `const` and only changing it to `let` if absolutely necessary.  (Your other `let`s in this function could also have been `const`s.)